docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -148,4 +148,12 @@ You will refer to this file later when configuring the plugin.
   more practical certificate lifetime, e.g., `--ttl=8760h` to export a one-year
   certificate.
 
+  Using impersonated credentials to issue the certificates requires you are
```

Comment:
1. Is this only true for impersonated credentials, or all the time?
2. I think we can improve the language here. How about the following?

If you are using impersonated credentials, you cannot issue certificates
that are valid longer than your existing credentials. For example, to issue
certificates with a `1000h` TTL, you must be logged in with a session that is
valid for at least 1000 hours. This means your user must have a role allowing
a `max_session_ttl` of at least 1000 hours, and you must specify a `--ttl`
when logging in:

```
$ tsh login --proxy=teleport.example.com --ttl=1001h
```

Reply:
it would be for your own too.  How about adding this? "or for your own user,"

If you are using impersonated credentials or for your own user, you cannot issue certificates
that are valid longer than your existing credentials. For example, to issue
certificates with a 1000h TTL, you must be logged in with a session that is
valid for at least 1000 hours. This means your user must have a role allowing
a max_session_ttl of at least 1000 hours, and you must specify a --ttl
when logging in:

Reply:
That doesn't read correctly to me. To avoid overcomplicating things, I recommend the following:

Note that you cannot issue certificates that are valid longer than your existing credentials. 
For example, to issue certificates with a 1000h TTL, you must be logged in with a session that is
valid for at least 1000 hours. This means your user must have a role allowing
a max_session_ttl of at least 1000 hours, and you must specify a --ttl
when logging in:

```
$ tsh login --proxy=teleport.example.com --ttl=1001h
```